examples/fedsimclr_example/fedsimclr_finetuning_example/client.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+from pathlib import Path
+from typing import Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+import torchvision.transforms as transforms
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from examples.models.cnn_model import SslEncoder, SslPredictionHead, SslProjectionHead
+from fl4health.clients.basic_client import BasicClient
+from fl4health.model_bases.fedsimclr_base import FedSimClrModel
+from fl4health.utils.dataset import TensorDataset
+from fl4health.utils.load_data import get_cifar10_data_and_target_tensors, split_data_and_targets
+from fl4health.utils.metrics import Accuracy
+
+
+def get_finetune_dataset(data_dir: Path, batch_size: int) -> Tuple[DataLoader, DataLoader]:
+    data, targets = get_cifar10_data_and_target_tensors(data_dir, False)
```

Comment:
very minor but maybe we keyword the train argument here to make it explicit that the boolean is saying load the test data? That is, `train=False`.